dts/arm/renesas/smartbond/da14699.dtsi

```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include "da1469x.dtsi"
+
+&sram0 {
+	reg = <0x20000200 (DT_SIZE_K(512) - 0x200)>;
```

Comment:
Why? Maybe you should add a note to make people aware of it.

Reply:
first 512 bytes of sram have to be reserved because of how memory remapping is configured after reset, but I managed to change this to be handled in linker script so dt regs are not confusing anymore